GitWorkflow-JFarr/GitWorkflow-JFarr/Program.cs

```diff
@@ -15,6 +15,7 @@ static class Program
         static void Main()
         {
             // This is a comment made by JFarr
+            // This is a comment made by JFarr in the first dev branch
             Application.EnableVisualStyles();
```

Comment:
This looks like the comment I expected.  It's missing punctuation though.. but that's not critical. -jfarr